File: android/src/main/java/group/lamantin/yandex/payment/Payment.kt
package group.lamantin.yandex.payment

import ru.yoo.sdk.kassa.payments.PaymentMethodType
import ru.yoo.sdk.kassa.payments.SavePaymentMethod

/**
 * @since 2019
 * @author Anton Vlasov - whalemare
 */
data class Payment(
  val amount: Double,
  val currency: String,
  val types: Set<PaymentMethodType>,
  val savePaymentMethod: SavePaymentMethod,
  val yooKassaClientId: String
)

File: ios/MyFramework/Payment.swift
import Foundation
import YooKassaPayments
import YooKassaPaymentsApi

struct Payment {
    let amount: Double
    let currency: Currency
    let types: Set<YooKassaPayments.PaymentMethodType>
    let savePaymentMethod: YooKassaPayments.SavePaymentMethod
    let moneyAuthClientId: String
}
